### PR TITLE
Tickets/dm34847: Added AuxTel Stop test report

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: QueryEfd.py [hunk 1]
```diff
@@ -250,10 +250,20 @@ def verify_topic(self, csc: str, topic: str) -> None:
...
         topic : `str`
             The name of the topic.
+        fields : `list`
+            The list of fields for the given topic.
+        expected_value : `str`
+            The expected value of the attribute.
         """
         csc, index = self._split_indexed_csc(csc)
-        topic_df = self.get_recent_samples(csc, topic, "*", 1, index)
+        attribute = fields[0]
```
Comment (mareuter) [inline QueryEfd.py:259]: Are there any cases where there's more than one attribute being passed here? If there are, how do you check the others? If there's only ever one attribute, then this isn't necessary and the parameters shouldn't be passed as list.

**Review by mareuter — COMMENTED** [review]
Want to understand a bit better the attribute check change.

File: Verify_AT_Stop.robot [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Force Tags    auxtel_stop
+
+*** Variables ***
+@{azimuthInPosition}    inPosition
+@{allAxesInPosition}    inPosition
+@{atMountState}    state
+
+*** Test Cases ***
+Verify ATDome logevent_azimuthInPosition
+    [Tags]    disabled
```
[the remark this answers is not in the document]
Comment (rbovill) [reply Verify_AT_Stop.robot:18]: Copy-paste error that I forgot to fix.  Fixed now.

File: QueryEfd.py (hunk 1 above)
Comment (rbovill) [reply QueryEfd.py:262]: Made some changes and added documentation to improve this keyword.

**Review by mareuter — APPROVED** [review]
Looks good to me.

**Pull request merged**